diff -up chromium-140.0.7339.80/third_party/swiftshader/third_party/llvm-16.0/BUILD.gn.me chromium-140.0.7339.80/third_party/swiftshader/third_party/llvm-16.0/BUILD.gn
--- chromium-140.0.7339.80/third_party/swiftshader/third_party/llvm-16.0/BUILD.gn.me	2025-09-04 06:45:21.570780130 +0200
+++ chromium-140.0.7339.80/third_party/swiftshader/third_party/llvm-16.0/BUILD.gn	2025-09-04 06:55:56.278501043 +0200
@@ -1599,6 +1599,9 @@ swiftshader_llvm_source_set("swiftshader
     "llvm/lib/Target/PowerPC/PPCVSXFMAMutate.cpp",
     "llvm/lib/Target/PowerPC/PPCVSXSwapRemoval.cpp",
     "llvm/lib/Target/PowerPC/TargetInfo/PowerPCTargetInfo.cpp",
+    "llvm/lib/MC/MCXCOFFObjectTargetWriter.cpp",
+    "llvm/lib/MC/MCAsmInfoXCOFF.cpp",
+    "llvm/lib/MC/MCInstrInfo.cpp",
   ]
 }
 
