Enable widevine_cdm on arm64

diff -up chromium-147.0.7727.137/third_party/widevine/cdm/widevine.gni.me chromium-147.0.7727.137/third_party/widevine/cdm/widevine.gni
--- chromium-147.0.7727.137/third_party/widevine/cdm/widevine.gni.me	2026-04-29 18:40:15.060405982 +0200
+++ chromium-147.0.7727.137/third_party/widevine/cdm/widevine.gni	2026-04-29 18:42:07.060690905 +0200
@@ -27,7 +27,8 @@ if (is_chromeos && !is_chromeos_device)
 library_widevine_cdm_available =
     (is_chromeos &&
      (target_cpu == "x64" || target_cpu == "arm" || target_cpu == "arm64")) ||
-    (target_os == "linux" && target_cpu == "x64") ||
+    (target_os == "linux" &&
+     (target_cpu == "x64" || target_cpu == "arm64")) ||
     (target_os == "mac" && (target_cpu == "x64" || target_cpu == "arm64")) ||
     (target_os == "win" &&
      (target_cpu == "x86" || target_cpu == "x64" || target_cpu == "arm64"))
