diff -up chromium-149.0.7827.53/build/config/compiler/BUILD.gn.me chromium-149.0.7827.53/build/config/compiler/BUILD.gn
--- chromium-149.0.7827.53/build/config/compiler/BUILD.gn.me	2026-04-06 11:35:59.082476460 +0200
+++ chromium-149.0.7827.53/build/config/compiler/BUILD.gn	2026-04-06 11:36:52.267527294 +0200
@@ -617,7 +617,6 @@ config("compiler") {
     # https://crbug.com/484082200 for background and https://crrev.com/c/7593035
     # for discussion.
     if (!is_wasm) {
-      cflags += [ "-fno-lifetime-dse" ]
     }
 
     # TODO(hans): Remove this once Clang generates better optimized debug info
diff -up chromium-149.0.7827.53/build/config/compiler/BUILD.gn.me chromium-149.0.7827.53/build/config/compiler/BUILD.gn
--- chromium-149.0.7827.53/build/config/compiler/BUILD.gn.me	2026-05-31 12:18:05.287133288 +0200
+++ chromium-149.0.7827.53/build/config/compiler/BUILD.gn	2026-05-31 12:19:16.908256575 +0200
@@ -588,12 +588,6 @@ config("compiler") {
   if (is_clang) {
     # Flags for diagnostics.
     cflags += [ "-fcolor-diagnostics" ]
-    if (!is_win) {
-      cflags += [ "-fdiagnostics-show-inlining-chain" ]
-    } else {
-      # Combine after https://github.com/llvm/llvm-project/pull/192241
-      cflags += [ "/clang:-fdiagnostics-show-inlining-chain" ]
-    }
     if (diagnostics_print_source_range_info && !is_win) {
       cflags += [ "-fdiagnostics-print-source-range-info" ]
     }
diff -up chromium-149.0.7827.53/build/config/sanitizers/sanitizers.gni.me chromium-149.0.7827.53/build/config/sanitizers/sanitizers.gni
--- chromium-149.0.7827.53/build/config/sanitizers/sanitizers.gni.me	2026-05-31 12:20:07.389243564 +0200
+++ chromium-149.0.7827.53/build/config/sanitizers/sanitizers.gni	2026-05-31 12:20:43.632843203 +0200
@@ -540,7 +540,6 @@ template("ubsan_hardening") {
         # be usable even in release builds, i.e. as widely as possible.
         # It's important not to have full-on UBSan workarounds activate
         # just because we built support for a specific sanitizer.
-        "-fsanitize-ignore-for-ubsan-feature=${invoker.sanitizer}",
       ]
       if (defined(invoker.cflags)) {
         cflags += invoker.cflags
